src/ssl.c

```diff
@@ -46277,17 +46277,30 @@ int wolfSSL_BN_is_odd(const WOLFSSL_BIGNUM* bn)
 
 /* return compliant with OpenSSL
  *   1 if BIGNUM is word, 0 else */
-int wolfSSL_BN_is_word(const WOLFSSL_BIGNUM* bn, WOLFSSL_BN_ULONG w)
+int wolfSSL_BN_is_word(const WOLFSSL_BIGNUM* bn, unsigned long w)
```

Comment:
Leave it as type WOLFSSL_BN_ULONG now the typedef has changed?

Reply:
Thanks for catching that!  I had already fixed that in bn.h (in #3522 ) but not in ssl.c.